$NetBSD: patch-aa,v 1.11 2007/09/14 12:03:37 joerg Exp $

Hack around GNUlib idiosyncracy where the overwritten stdio.h includes
sys/types.h which includes stdint.h (overwritten!) which includes wchar.h
which includes stdio.h, BOOM.

--- lgl/stdio_.h.orig	2007-09-14 11:29:05.000000000 +0000
+++ lgl/stdio_.h
@@ -35,6 +35,14 @@
 #include <stdarg.h>
 #include <stddef.h>
 
+#ifdef __DragonFly__
+#include <machine/stdint.h>
+#ifndef _SSIZE_T_DECLARED
+#define _SSIZE_T_DECLARED
+typedef __ssize_t	ssize_t;
+#endif
+#endif
+
 #if (@GNULIB_FSEEKO@ && @REPLACE_FSEEKO@) \
   || (@GNULIB_FTELLO@ && @REPLACE_FTELLO@) \
   || (@GNULIB_GETDELIM@ && !@HAVE_DECL_GETDELIM@) \
